package.json

```diff
@@ -47,19 +46,12 @@
 			"last 1 safari version"
 		]
 	},
-	"babel": {
-		"presets": [
-			"react-app"
-		]
-	},
 	"dependencies": {
```

Comment:
some of the deps were moved in dev deps, mainly because of:
```
The difference between these two, is that devDependencies are modules which are only required during development, while dependencies are modules which are also required at runtime.
```
more info: https://medium.com/@dylanavery720/npmmmm-1-dev-dependencies-dependencies-8931c2583b0c